charms/worker/k8s/src/charm.py

```diff
@@ -95,6 +95,7 @@ def _cluster_departing_unit(event: ops.EventBase) -> Union[Literal[False], ops.U
         isinstance(event, ops.RelationDepartedEvent)
         and event.relation.name in ["k8s-cluster", "cluster"]
         and event.departing_unit
+        or False
```

Comment:
`event.departing_unit` grew a new return type of `None`.  And since it did -- the return typing changed to include a `NoneType`